src/moi_oracle.jl

```diff
@@ -232,17 +232,17 @@ function compute_inface_extreme_point(
 	kwargs...,
 ) where {OT, T <: Real}
 	n = size(direction, 1)
-	a = compute_inface_extreme_point(lmo, vec(direction), x)
+	a = compute_inface_extreme_point(lmo, vec(direction), vec(x))
 	return reshape(a, n, n)
 end
 
 # Fast way to compute gamma_max.
 # Check every constraint and compute the corresponding gamma_upper_bound. 
-function dicg_maximum_step(lmo::MathOptLMO{OT}, direction, x) where {OT}
+function dicg_maximum_step(lmo::MathOptLMO{OT}, direction, x;tol=1e-6) where {OT}
```

Comment:
It would be good to have a doc string here, i.e.
```
"""
Description of the function.
"""
```
This would also be good for the `compute_inface_extreme_point` as well as `is_inface_feasible` function.